### PR TITLE
DEV-1057 Migrate AvatarImageFallback component to ui-shared-components 

### PR DESCRIPTION
**Pull request by SKona306**
https://offtherecord.atlassian.net/browse/DEV-1057

imagefallback

**Migrated avatar image fallback component to ui-shared-components library**

Bindings: 
- **src:** profileImageUrl (string) optional
- **avatar-class:** CSS class name (string) optional
- **name:** (string) optional

**Areas of opportunity:**

imagefallback2

- Note the function above looks for the value of `default-user`  in the src; it may be better to deal with this in the parent application instead of the component.

### CODE REVIEW

File: app/components/app-avatar-image-fallback.component.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    text-align: center;
+    height: 62px;
+    width: 62px;
+    border-radius: 50%;
+    font-size: 39px;
+    display: flex !important;
+    justify-content: center;
+    align-items: center;
+    color: #FFFFFF;
+}
```
Comment (JStein92) [inline app/components/app-avatar-image-fallback.component.scss]: lets give these BEM style classnames so they don't have any potential conflicts. I could especially see "fallback" being a classname in another app at some point.

File: app/components/app-toggle-card.component.scss [hunk 2]
```diff
@@ -5,6 +5,8 @@
 }
 
 .app-toggle-card {
+    cursor: none;
```
Comment (JStein92) [inline app/components/app-toggle-card.component.scss:8]: what's this change about?

File: app/components/avatar-image-fallback/avatar-image-fallback.component.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        });
+    }
+
+    isProfilePictureShowing() {
+        if (!this.src || this.src.includes('default-user')) {
+            return false;
+        } else {
+            return true;
+        }
+    }
```
Comment (JStein92) [inline app/components/avatar-image-fallback/avatar-image-fallback.component.ts]: let's take this logic out - this component should be agnostic to content inside the src. It'll be up to the parent components to handle this.

File: index-test.html [hunk 4]
```diff
@@ -259,16 +282,42 @@ <h3>Toggle Card</h3>
...
+            src="facebookProfileImage"
+            avatar-class="customer"
+            name="Earl Benz">
+        </avatar-image-fallback>
+        <!-- use case: img tags that throw an error -->
+        <span>
+          <avatar-image-fallback
+              src="brokenProfileImageUrl"
+              name="Shaun Kent"></avatar-image-fallback>
+        </span>
```
Comment (JStein92) [inline index-test.html]: nicely done!

**Review by JStein92 — CHANGES_REQUESTED** [review]
Really great PR! Appreciate the description and images in the pr summary. And the code is clean. Just a few comments!

File: app/components/app-toggle-card.component.scss (hunk 2 above)
Comment (SKona306) [reply app/components/app-toggle-card.component.scss:8]: the toggle card in lawfirm app has a cursor when you hover over the entire vacation mode card but the card is not clickable. the only part of the toggle card that should have a cursor is the toggle.

File: app/components/app-avatar-image-fallback.component.scss [hunk 5]
```diff
@@ -9,7 +9,7 @@
     width: 72px;
 }
 
-.app-avatar-image {
+.app-avatar-image__default-user {
```
Comment (JStein92) [inline app/components/app-avatar-image-fallback.component.scss]: app-avatar-image-fallback__default-user

**Review by JStein92 — APPROVED** [review]
approved! Just tweak the BEM name a bit so it fits the component

**Review by JStein92 — APPROVED** [review]
Nice! @Morkos you're up. Lower priority but this will be great to have.

**Review by Morkos — APPROVED** [review]
Looking good!

**Pull request merged**